A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -271,20 +287,24 @@ class FMCMainDisplay extends BaseAirliners {
         }
         SimVar.SetSimVarValue("L:AIRLINER_MCDU_CURRENT_FPLN_WAYPOINT", "number", this.currentFlightPlanWaypointIndex);
     }
+
     get cruiseFlightLevel() {
         return this._cruiseFlightLevel;
     }
+
     set cruiseFlightLevel(fl) {
-        this._cruiseFlightLevel = Math.round(fl / 5) * 5;
+        this._cruiseFlightLevel = Math.round(fl);
```

Comment:
**question**: I can understand why rounding is needed when doing division. However, if `fl` is always an integer rounding is no longer necessary. Did you check whether or not it is?

Reply:
Decimal numbers can be entered and even if the entry appears as an integer after the fact, it seems to be internally stored as a decimal value still, and only formatted as an integer. This ensures the stored value is an integer.